apps/ocpp/consumers/base/consumer.py

```
@@ -1295,6 +1295,8 @@ async def _handle_data_transfer(
         else:
             vendor_message_id_text = str(vendor_message_id)
         connector_value = self.connector_value
+        if connector_value is None and connector_hint is not None:
```

Ruff (0.14.14)

[error] Undefined name `connector_hint`

(F821)